mlos_bench/mlos_bench/tests/optimizers/toy_optimization_loop_test.py

```diff
@@ -113,7 +113,7 @@ def test_smac_optimization_loop(mock_env_no_noise: MockEnv, smac_opt: MlosCoreOp
         "vmSize": "Standard_B2s",
         "idle": "mwait",
         "kernel_sched_migration_cost_ns": 297669,
-        "kernel_sched_latency_ns": 290365100,
+        "kernel_sched_latency_ns": 290365137,
```

Comment:
Shouldn't this be quantized?

Reply:
I wish! (and the quantized value should be 300000000). Apparently, none of our optimizers actually samples ConfigSpace for new values directly (as it should) and instead just picks the argmax of the acquisition function and returns this as the new suggestion. That still does not explain the change in the value here, but it is consistent across all branches (main included). Could be a SMAC update?

Reply:
Hmm, when I was testing this with the older version of SMAC we were getting quantized values back.  I think that's the behavior we should expect to see after the monkey patching hack happens, right?